components/annot-discrete-choices.vue

```diff
@@ -173,8 +189,12 @@
 
                         if ( this.relevantColumns.includes(columnName) ) {
 
-                            // TODO: If "value" is a missing value or doesn't fit the heuristic, this will currently break!
-                            transformedTable[index][columnName] = this.transformedValue(columnName, transformedTable[index][columnName]);
+                            if ( this.isMissingValue(columnName, transformedTable[index][columnName]) ) {
+                                // TODO: this string should be replaced by an app-wide way to designate missing values
+                                transformedTable[index][columnName] = "missing value";
```

Comment:
Typically, it's good to remove hardcoded strings from logic and place them somewhere central (i.e. the `uiText` object in our components).

It's possible something like this could be placed in the store in some common string value list for retrieval and conditional checks.